src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -550,8 +551,7 @@ define(function (require, exports, module) {
                         }
                     }, this)
                     .map(function (layer, visibleIndex) {
-                        var isDragTarget = dragTargets && dragTargetSet.has(layer),
-                            isDropTarget = dropTarget && dropTarget.key === layer.key,
+                        var isDropTarget = !!dropTarget && dropTarget.key === layer.key,
```

Comment:
This isn't really a nit, but I don't think it's actually necessary to convert the first parameter into a boolean here.